src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+import java.util.function.Consumer;
+
+import duke.commands.Command;
+import duke.task.TaskList;
+
+/**
+ * The Duke chatbot
+ */
+public class Duke {
+    private final TaskList tasks;
```

Comment:
For constants, perhaps you may want to consider having them in full caps (so that it is easier to distinguish)